boilerplate/app/navigators/navigation-utilities.tsx

```diff
@@ -39,6 +39,9 @@ export function getActiveRouteName(state: NavigationState | PartialState<Navigat
  * the navigation or allows exiting the app.
  */
 export function useBackButtonHandler(canExit: (routeName: string) => boolean) {
+  // ignore if iOS ... no back button!
+  if (Platform.OS === "ios") return
```

Comment:
Breaks a hooks rule, but since this will never change on the fly somehow, shouldn't be an issue.